Source/WebInspectorUI/UserInterface/Views/ComputedStyleSection.js

```diff
@@ -129,16 +130,38 @@ WI.ComputedStyleSection = class ComputedStyleSection extends WI.View
         this.needsLayout();
     }
 
+    get isEmpty()
+    {
+        this.updateLayoutIfNeeded();
+
+        return !this._propertyViews.length;
+    }
+
+    set propertiesGetter(fn)
+    {
+        console.assert(typeof fn === "function", fn, "The custom getter must be a function that returns an array of properties.");
+
+        this._propertiesGetter = fn;
```

Comment:
something about this feels icky

we already are creating a separate `WI.ComputedStyleSection` for each `WI.CSSStyleDeclaration.VariablesGroupType` (via `WI.ComputedStyleDetailsPanel.prototype._renderVariablesStyleSectionGroup`) , so could we instead pass the `WI.CSSStyleDeclaration.VariablesGroupType` to the `WI.ComputedStyleSection` and then have `WI.ComputedStyleSection.prototype.get properties` use it (if provided)?

Reply:
I like this idea. Implemented. Thanks!